File: backend/models/manual_geolocation.go
package models

type ManualGeolocation struct {
	Uuid                 string  `gorm:"primary_key" json:"uuid"`
	ClassifiedPeriodUuid string  `json:"classifiedPeriodUuid"`
	Latitude             float64 `json:"latitude"`
	Longitude            float64 `json:"longitude"`
	CreatedOn            int64   `json:"createdOn"`
	DeletedOn            int64   `json:"deletedOn"`
	UserId               int64
}

File: core_backend/core_backend/Models/ManualGeolocation.cs
using System.ComponentModel.DataAnnotations;

namespace core_backend.Models
{
    public class ManualGeolocation
    {
        [Key]
        public string Uuid { get; set; }
        public string ClassifiedPeriodUuid { get; set; }
        public float Latitude { get; set; }
        public float Longitude { get; set; }
        public long CreatedOn { get; set; }
        public long DeletedOn { get; set; }
        public string UserId { get; set; }
    }
}
